app/navigation/navigation-core/src/main/kotlin/com/hedvig/android/navigation/core/HedvigDeepLinkContainer.kt

```diff
@@ -150,4 +156,5 @@ val HedvigDeepLinkContainer.allDeepLinkUriPatterns: List<String>
     conversation.first(),
     travelAddon.first(),
     travelCertificate.first(),
+    changeTierWithoutContractId.first(),
```

Comment:
```suggestion
    changeTierWithoutContractId.first(),
    changeTierWithContractId.first(),
```
I think it'd then be able to match both, but I'd need to test to be sure. In any case I think it should be safe to commit this change anyway

Reply:
yes, absolutely, thanks for catching